webapp/packages/core-root/src/SessionResource.ts

```diff
@@ -95,6 +92,23 @@ export class SessionResource extends CachedDataResource<SessionState | null> {
     return session;
   }
 
+  touchSession = () => {
+    if (this.touchSessionTimer) {
+      return;
+    }
+
+    if (this.data?.valid) {
+      this.graphQLService.sdk.touchSession();
```

Comment:
This call will update the session lifetime. We need to read the session from this method and update `setData` of the resource like it was in `refreshSilent`